### PR TITLE
GEOD-306 fix server code to map and store problem sources

### PR DESCRIPTION
**Pull request by heyagosper**

### CODE REVIEW

**Comment by oehm-smith** [comment]
:+1: :m:

File: gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      </addColumn>
+      <addColumn tableName="sitelog_mutlipathsource">   
+         <column name="notes" remarks="any extra comments about the problem" type="VARCHAR(4000)"/>
+      </addColumn>
+   </changeSet>
+   
+   <changeSet author="heya"
+              id="1490933518333-2"
+              objectQuotingStrategy="QUOTE_ALL_OBJECTS">
+      <addColumn tableName="sitelog_signalobstraction">
```
Comment (lbodor) [inline gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml]: Spelling: obstruction

File: gws-core/src/main/resources/db/geodesy-database-changelog.xml [hunk 2]
```diff
@@ -1,51 +1,58 @@
...
+	<include file="db/add-table-column-comments-part2.xml" />
+	<include file="db/add-table-column-comments-part3.xml" />
+	<include file="db/add-delete-column-in-sitelog.xml" />
+	<include file="db/add-user-registration.xml"/>
     <include file="db/add-user-registration-remarks.xml"/>
-    <include file="db/rename-local-episodic-event-table.xml"/>
-    <include file="db/add-sitelog_site_position_points.xml"/>
+    <include file="db/rename-local-episodic-event-table.xml" />
+    <include file="db/add-sitelog_site_position_points.xml" />
+    <include file="db/add-missing-columns-to-problem-source-tables.xml"/>
```
Comment (lbodor) [inline gws-core/src/main/resources/db/geodesy-database-changelog.xml]: Remove tabs please.

File: gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+         <column name="notes" remarks="any extra comments about the problem" type="VARCHAR(4000)"/>
+      </addColumn>
+   </changeSet>
+   
+   <changeSet author="heya"
+              id="1490933518333-2"
+              objectQuotingStrategy="QUOTE_ALL_OBJECTS">
+      <addColumn tableName="sitelog_signalobstraction">       
+         <column name="possible_problem_source" remarks="the source of the problem" type="VARCHAR(4000)"/>   
+      </addColumn>
```
Comment (lbodor) [inline gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml]: We are using TEXT for strings, not VARCHAR.

File: gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml (hunk 1 above)
Comment (heyagosper) [inline gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml]: that's the table name at the moment. i want to fix that in a separate PR

Comment (heyagosper) [inline gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml]: i could just do it in this one really

**Comment by heyagosper** [comment]
I guess I do have to change some source code as well

File: gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml (hunk 1 above)
Comment (lbodor) [inline gws-core/src/main/resources/db/add-missing-columns-to-problem-source-tables.xml]: Up to you, separate PR is fine.

**Comment by lbodor** [comment]
The build errors are resolved now that the xml types have been regenerated on master of ogc-schemas.

**Pull request merged**